.github/workflows/build.yml

```diff
@@ -0,0 +1,16 @@
+name: Build
+on: 
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    container:
+      image: rancher/dapper:v0.6.0
+    steps:
+      - name: Add Git
+        run: apk add -U git
+      - name: Checkout Code
+        uses: actions/checkout@v4
+      - name: Run Dapper CI
+        run: dapper ci
```

Comment:
Looking at the drone configuration, I think docker build (dry run) is also part of `pull_request` so we may want to add it here. Though it is not part of `push` or `tag`.